hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/InitializeEntityBalanceMigration.java

```diff
@@ -87,4 +89,12 @@ protected void doMigrate() {
         int count = jdbcOperations.update(INITIALIZE_ENTITY_BALANCE_SQL);
         log.info("Initialized {} entities balance in {}", count, stopwatch);
     }
+
+    @EventListener
+    public void reRunMigration(InitializeBalanceEvent event) {
```

Comment:
we don't want to run it when the transaction in which the event is published fails

```suggestion
   @TransactionalEventListener(classes = InitializeBalanceEvent.class)
    public void reRunMigration() {
```